packages/auth/src/auth.ts

```diff
@@ -111,7 +111,7 @@ class Auth implements IAuth {
     };
     let authPlugin;
     try {
-      authPlugin = new HTPasswd(plugingConf, pluginOptions);
+      authPlugin = new HTPasswd(plugingConf, pluginOptions as any as PluginOptions<HTPasswdConfig>);
```

Comment:
This was necessary as running `pnpm run build` would generate a TypeScript error now that the types are properly defined in the `htpasswd` auth plugin.